systemtests/tests/python-bareos/test_acl.py

```diff
@@ -402,3 +398,30 @@ def test_status_subscription_admin(self):
     def test_status_subscription_user_fails(self):
         with self.assertRaises(bareos.exceptions.JsonRpcErrorReceivedException):
             self._test_status_subscription("client-bareos-fd", "secret")
+
+    def test_limited_command_acl(self):
+        """
+        The console "limited-operator" uses the profile "operator",
+        but disallows the command ".consoles".
+        """
+        logger = logging.getLogger()
+
+        username = "limited-operator"
+        password = "secret"
+
+        console = bareos.bsock.DirectorConsoleJson(
+            address=self.director_address,
+            port=self.director_port,
+            name=username,
+            password=password,
+            **self.director_extra_options
+        )
+
+        with self.assertRaises(bareos.exceptions.JsonRpcErrorReceivedException):
+            result = console.call(".consoles")
+            # logger.debug(result)
+            # consoles = [item["name"] for item in result["consoles"]]
+            # self.assertIn(username, consoles)
+
+        with self.assertRaises(bareos.exceptions.JsonRpcErrorReceivedException):
+            result = console.call(".consol")
```

Comment:
i feel like we are missing something here.  What is this trying to test ?  I would assume that we would test that these calls return nothing as the acl disallows these commands but im not seeing this here.

We should probably also test the opposite: Have a positive acl and check that it actually allows us to use the commands.

Reply:
DirectorConsoleJson will raise an exception, if it detects an error. I added a comment for clarification to the code.